std/unicode.zig

```diff
@@ -155,6 +156,18 @@ pub const Utf8View = struct {
         return initUnchecked(s);
     }
 
+    pub fn eql(self: &const Utf8View, other: &const Utf8View) bool {
+        return mem.eql(u8, self.bytes, other.bytes);
+    }
+
+    pub fn slice(self: &const Utf8View, start: usize, end: usize) !Utf8View {
```

Comment:
I would expect that slicing a unicode string would be from `start_code_point` to `end_code_point`, and not just slice the memory and validate that the substring still is valid.

Reply:
But then again, the string could be big so maybe this is better

Reply:
Yes that check is not needed :), as said in my other response.  This slice will be renamed because that is a valid point it mainly is used in my split function for example which doesn't need a concept of end points; how about sliceRaw?  I'll do a proper slice one since I can just check the first bit to detect when to iterate the index though it'll be max O(N) { N := the raw point corresponding to the code point index } just to detect the range which is unavoidable.

Reply:
Somehow I missed your second point, it isn't always needed :).  So I'll probably add it as 'sliceRawUnchecked' for example